docs_app/content/tutorial/8-subjects.md

```diff
@@ -0,0 +1,304 @@
+# 8. Subjects
+
+## TLDR:
+
+- Both an [observer](GL) and an [observable](GL). Inheriting from [`Observable`](API).
+- The primary use case is [multicasting](GL).
+- The next use case is eventing, such as dealing with single-function event handlers in frameworks like React and Angular.
```

Comment:
I have been using Subject heavily for reacting to application actions. For example, combining data streams (from an http get) with a subject to perform actions such as filtering or selection (finding one in the retrieved list), etc. That doesn't seem to be included here? It does not appear to be the same as "event handlers"?

Also, this scenario:
```
products$ = combineLatest([
  this.filterAction$,
  this.currentPage$,
  this.pageSizeAction$
]).pipe(
    switchMap(([filter, pageNumber, pageSize]) =>
      this.http.get(this.productsUrl, {
        params: {
          filter: filter,
          page: pageNumber.toString(),
          limit: pageSize.toString() 
        }
      }))
   );
```

I know the above is to complex and Angular specific, but this general idea of "reacting to application actions" may be a worthwhile use case to discuss here?

Where each of the combined streams are a subject.